app/assets/stylesheets/common/base/modal.scss

```diff
@@ -621,3 +623,35 @@
   position: absolute;
   width: 95%;
 }
+
+.modal {
+  &.has-panels {
+    .panel-tabs {
```

Comment:
not happy with the naming so far "panel-tab" "share-panel" "invite-panel"... will make it more consistent